job-templates/kubernetes_release.json

```diff
@@ -0,0 +1,85 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "orchestratorRelease": "1.13"
```

Comment:
why is the release tagged as 1.13 here? i of course don't understand this json file well